ruleapi/tests/common.go

```diff
@@ -60,3 +69,66 @@ type txnCtx struct {
 	Testing *testing.T
 	TxnCnt  int
 }
+
+func CaptureOutput(f func()) string {
```

Comment:
@LakshmiMekala I feel like CaptureStdOutput() would be more appropriate name than CaptureOutput() as it is capturing standard output.

Reply:
required changes are done